tests/operator/operator.go

```diff
@@ -2443,21 +2454,23 @@ spec:
 
 			}, 240*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
 
-			// wait for virt-api and virt-controller to pick up the change that CDI no longer exists.
-			time.Sleep(30 * time.Second)
+			// wait for virt-api and virt-controller to pick up the change that CDI no longer exists & stabilize
+			time.Sleep(10 * time.Second)
```

Comment:
Do we want to do something like
```
Wait for not ready
Wait for ready
```
? Not realizing on knowing the internal timeout?